receiver/statsdreceiver/transport/client/client.go

```diff
@@ -7,33 +7,25 @@ import (
 	"fmt"
 	"io"
 	"net"
+	"strings"
 )
 
 // StatsD defines the properties of a StatsD connection.
 type StatsD struct {
-	Host string
-	Port int
-	Conn io.Writer
+	transport string
+	address   string
+	Conn      io.Writer
```

Comment:
Could we un-export this as well?

Reply:
<img width="300" alt="Screenshot 2023-08-03 at 09 44 51" src="https://github.com/kang-makes/opentelemetry-collector-contrib/assets/43335750/d1711e49-0261-46c6-9eaf-1504e5d65abf">
 
I see used only here

Reply:
I did not change it as it was a lib used for testing and it was already like that. But as I am refactoring it, I have no issue with changing it. Unexported :)